main.go

```diff
@@ -48,6 +47,11 @@ func main() {
 			EnvVar: "PLUGIN_TEMPLATE",
 		},
 		cli.StringFlag{
+			Name:   "image_url",
+			Usage:  "slack image url",
+			EnvVar: "PLUGIN_IMAGE_URL",
+		},
```

Comment:
let's use `image` instead of `image_url` as the command line argument. No need to include the `_url` from the command line.

```diff
 		cli.StringFlag{
 -			Name:   "image_url",
 +			Name:   "image",
 			Usage:  "slack image url",
 			EnvVar: "PLUGIN_IMAGE_URL",
 		},
```